src/coro/Scheduler.zig

```diff
@@ -98,6 +98,20 @@ pub fn tick(self: *@This(), mode: aio.Dynamic.CompletionMode) aio.Error!usize {
     } else {
         _ = try self.io.complete(mode);
     }
+    if (self.running.first) |first| {
```

Comment:
This won't work if the scheduler is blocking on IO and cycling all the tasks is not something I want to do here.